### ISSUE TITLE
MT7610E: TX-Power Values incorrect -> low range

### ISSUE THREAD

**Issue by MoritzThome** [openwrt/mt76#604]
My TP-Link Vr200v uses MT7610E for 5GHz. In Stock firmware range is good (signal should be 22dbm), in latest master it is very limited:
`Frequencies:
			* 5180 MHz [36] (12.0 dBm)
			* 5200 MHz [40] (12.0 dBm)
			* 5220 MHz [44] (12.0 dBm)
			* 5240 MHz [48] (12.0 dBm)
			* 5260 MHz [52] (12.0 dBm)
			* 5280 MHz [56] (12.0 dBm)
			* 5300 MHz [60] (12.0 dBm)
			* 5320 MHz [64] (12.0 dBm)
			* 5500 MHz [100] (14.0 dBm)
			* 5520 MHz [104] (14.0 dBm)
			* 5540 MHz [108] (14.0 dBm)
			* 5560 MHz [112] (14.0 dBm)
			* 5580 MHz [116] (14.0 dBm)
			* 5600 MHz [120] (15.0 dBm)
			* 5620 MHz [124] (15.0 dBm)
			* 5640 MHz [128] (15.0 dBm)
			* 5660 MHz [132] (15.0 dBm)
			* 5680 MHz [136] (15.0 dBm)
			* 5700 MHz [140] (15.0 dBm)
			* 5720 MHz [144] (16.0 dBm)
			* 5745 MHz [149] (16.0 dBm)
			* 5765 MHz [153] (16.0 dBm)
			* 5785 MHz [157] (17.0 dBm)
			* 5805 MHz [161] (17.0 dBm)
			* 5825 MHz [165] (18.0 dBm)
			* 5845 MHz [169] (disabled)
			* 5865 MHz [173] (disabled)
`

This Problem is not new and should be fixed since a lot of people are affected with this bug.
I tried to change the val manually (in eeprom.c), but had no luck, this brings no difference.
If i can provide helpful information please let me know! Thank you for your support!

**Comment by ArtemiisFowl** [comment]
> OFF-TOPIC: will the VDSL* part of that device work?
Yes, working great up to 100mbit vectoring connections. Only for long distances (bad snr) from router to dslam a broadcom chipset would be better if dslam is also broadcom (like most of them in germany)

**Comment by mrkiko** [comment]
Hello!

Can you try latest OpenWRt GIT HEAD and report back?
Thanks!
